packages/arb-token-bridge-ui/src/components/TransferPanel/TokenInfoTooltip.tsx

```diff
@@ -0,0 +1,93 @@
+import { ERC20BridgeToken } from '../../hooks/arbTokenBridge.types'
+import { useNetworks } from '../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+import { shortenAddress } from '../../util/CommonUtils'
+import { captureSentryErrorWithExtraData } from '../../util/SentryUtils'
+import { ExternalLink } from '../common/ExternalLink'
+import { Tooltip } from '../common/Tooltip'
+import { TokenLogo } from './TokenLogo'
+import { useNativeCurrency } from '../../hooks/useNativeCurrency'
+
+const createBlockExplorerUrlForToken = ({
+  explorerLink,
+  tokenAddress
+}: {
+  explorerLink: string | undefined
+  tokenAddress: string | undefined
+}): string | undefined => {
+  if (!explorerLink) {
+    return undefined
+  }
+  if (!tokenAddress) {
+    return undefined
+  }
+  try {
+    const url = new URL(explorerLink)
+    url.pathname += `token/${tokenAddress}`
+    return url.toString()
+  } catch (error) {
+    captureSentryErrorWithExtraData({
+      error,
+      originFunction: 'createBlockExplorerUrlForToken'
+    })
+    return undefined
+  }
+}
+
+export const TokenInfoTooltip = ({
+  token
+}: {
+  token: ERC20BridgeToken | null
+}) => {
+  const [networks] = useNetworks()
+  const { isDepositMode, childChainProvider } =
+    useNetworksRelationship(networks)
+  const childChainNativeCurrency = useNativeCurrency({
+    provider: childChainProvider
+  })
+
+  if (!token) {
+    return <span>{childChainNativeCurrency.symbol}</span>
+  }
+
+  const tokenAddress = isDepositMode ? token.address : token.l2Address
```

Comment:
we are looking for the destination address so this is wrong

currently on prod we don't show links for USDC because for Ethereum<->Arb One / Sepolia<->Arb Sepolia, users can use CCTP / arb bridge and the addresses for the two are different, might need to consider it on the design side

Reply:
Updated, we don't show the tooltip for USDC now either